core/dbt/parser/schemas.py

```diff
@@ -729,7 +738,9 @@ def patch_node_properties(self, node, patch: "ParsedNodePatch") -> None:
         node.columns = patch.columns
         node.name = patch.name
 
-        if not isinstance(node, ModelNode):
+        if isinstance(node, ModelNode):
+            node.time_spine = patch.time_spine
```

Comment:
I think the right place for this is actually in `ModelPatchParser.patch_constraints`: https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/parser/schemas.py#L905

Reply:
Updated!